robot_smach_states/src/robot_smach_states/util/designators/core.py

```diff
@@ -76,17 +76,6 @@ def _get_name(self):
     resolve_type = property(_get_resolve_type)
     name = property(_get_name)
 
-    def __str__(self):
```

Comment:
Why is it removed? Was it wrong?

Reply:
Mostly because it was a huge blob of text that was more confusing than useful.